frontend/src/index.html

```diff
@@ -7,6 +7,7 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/x-icon" href="favicon.ico">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```

Comment:
We could also get this through npm here: https://www.npmjs.com/package/typeface-montserrat
(Maybe this is worth discussing)